sys/linux/bpf_lsm.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/bpf.h>
+
+resource fd_bpf_lsm_prog[fd]
+resource btf_id[int32]
+
+bpf_lsm_func_names = "bpf_lsm_path_mkdir"
+
+syz_btf_id_by_name(name ptr[in, string[bpf_lsm_func_names]]) btf_id
+
+type bpf_insn_tmpl[CODE, DST_REG, SRC_REG, OFF, IMM] {
+	code	const[CODE, int8]
+	dst	const[DST_REG, int8:4]
+	src	const[SRC_REG, int8:4]
+	off	const[OFF, int16]
+	imm	const[IMM, int32]
+}
+
+# Test program 0:
```

Comment:
Might be good to document what this does, for if/when we start expanding the corpus of bpf lsm programs.